applications/spacewalk-bridge.md

```diff
@@ -41,7 +41,9 @@ The architecture of the bridge consists of the following components:
 
 - **Vaults**: this is a set of escrow accounts used to lock assets in Stellar. Their behavior is defined in XCLAIM and interBTC. In Spacewalk they have an additional property: each vault has an allow list of assets that it can lock and support for bridging operations between Stellar and the Substrate chain. This allow list is implemented through [trustlines](https://developers.stellar.org/docs/issuing-assets/anatomy-of-an-asset/#trustlines) of the Stellar account. There can be at most 1000 supported assets per vault due to limitations in Stellar. Stellar users initiate a deposit by sending tokens to an appropriate vault, which they request from the bridge pallet prior to the deposit. Likewise vaults will unlock and send tokens back to Stellar accounts during a withdrawal. Every vault needs to lock a certain amount of DOT or KSM (or related) tokens as collateral with the bridge pallet. These tokens are slashed in case the vault misbehaves.
 - **Bridge Pallet**: this is the main component of the Spacewalk bridge that implements all logic on the side of the Substrate-based chain. Its behavior is based on interBTC. It is particularly responsible for minting tokens during deposits and burning tokens during withdrawals. It is able to support any Stellar asset by employing the [Tokens](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/tokens) and [Currrencies](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/currencies) pallets of the Substrate Open Runtime Module Library. The storage of the bridge pallet maintains the complete state that is required for the bridge to work correctly. This state contains (among others): the account ids of the vaults, the asset allow lists of each vault and book keeping information about the state of the Stellar network.
-- **Stellar Oracle**: is a system that provides information about the state of the Stellar network to the bridge pallet. In interBTC this is implemented through a chain relay in the bridge pallet. However, we will aim for a solution where every node of the Substrate-based chain is co-located with a Stellar validator node, which it uses as an oracle.
+- **Stellar Oracle**: is a decentralized system that provides information about the state of the Stellar network to the bridge pallet. In interBTC this is implemented through a Bitcoin chain relay. However, for reasons explaned above we cannot implement a chain relay for Stellar in the same way. The Stellar oracle is trustless and reliable and its functionality is based on specific unique aspects of Stellar:
```

Comment:
I think you're mixing up things. In interBTC the [oracle](https://spec.interlay.io/spec/oracle.html) is a different and unrelated component from the [BTC relay](https://spec.interlay.io/spec/btc-relay/index.html). This also goes for the Stellar oracle, what you're describing is not an oracle but a relayer.

Reply:
On second look, it's evident you're not confusing the functionality but this is simply an issue with the naming. I'd suggest you consider renaming this component. The term 'oracle' is commonly used to describe any bridge from a blockchain to the real world, and I've never seen it used to mean a bridge to another blockchain or a relayer, light client etc. If you have a reference to anywhere this meaning is accepted feel free to point me to it.

Reply:
I should also add that regardless of whether you want to change this or not, I'm happy to accept your proposal.

Reply:
Thanks for your feedback. Indeed interBTC requires an oracle for price feeds in order to assess whether vault collaterals are sufficient and to determine that a vault needs to be liquidated. This is also required for our bridge and even at higher complexity level as we support a multitude of assets that the vaults can lock.

I agree that it is confusing that I was talking about a different oracle here. A very general definition of an oracle is a system (consisting of off-chain and on-chain components) that feeds data from the outside world into a pallet or smart contract and the on-chain logic would then verify and aggregate that information and find consensus on it. So in this liberal sense the component I refer to as Stellar oracle is indeed an oracle. I avoided the term relay because our component works differently to typical chain relays for blockchains that use Nakamoto consensus (I explain the differences in our proposals).

If this explanation could not resolve the confusion, I am happy to rename.

Reply:
No worries, that makes sense and @Noc2 also shared another example of where the term was used in this manner with me. I understand your reasoning and to be honest, I agree that logically it fits perfectly, but found using the same name for a different component in what is otherwise the same system confusing.